frontend/src/lib/components/LemonSelectWithSearch/LemonSelectWithSearch.tsx

```diff
@@ -0,0 +1,97 @@
+import { Select } from 'antd'
```

Comment:
Will importing Ant here impact the `@posthog/lemon-ui` package @mariusandra?

Reply:
I think we want to ultimately distance ourselves from antd. It's a great library to get started, but I'd love to see a world where `@posthog/lemon-ui` has no dependency on antd, and thus its exported bundle is a good megabyte smaller.

One way to helps us get there is by not building lemon components on top of ant components. :)

Reply:
@mariusandra agreed, long term we want to ditch it but replicating the functionality would take a fair amount of effort, hence the quick but pretty choice here 😉

Reply:
Sure, won't block for that. The end goal is still "react component exports", which will benefit greatly from removing antd.